src/cms/custom-control.scss

```diff
@@ -5,12 +5,16 @@
   border-top-right-radius: 5px;
 
   /*
-   * To override or revert Decap CMS styling that adversely affects
-   * Clue content styling in the editor, place related style rules
+   * To override or revert Decap CMS style rules that adversely affect
+   * CLUE content styling in the editor, place related style rules
    * within the .canvas .document-content block below.
    */
   .canvas .document-content {
     .text-tool-tile {
```

Comment:
Would it be slightly better if I removed the `.text-tile-tool` part here so it's not specific to the text tile?

I wasn't sure if that might cause the styles to affect other tiles in unexpected ways. I wouldn't think so and I didn't see any issues when I removed it, but it's possible I'm not aware of some particular feature of another tile that it would affect. I'm also not sure that would really address [the problem we'd like to avoid](https://github.com/concord-consortium/collaborative-learning/pull/1664#pullrequestreview-1369816567). It'd make the code less specific to the text tile, but I think ultimately we'd prefer not to have to style any CLUE tile content from here, right?

Reply:
Yeah I don't really know what is best because I don't remember how CSS rules are applied and how the rules work for which ones take precedence of the other.

My guess would be that the safest thing would be to apply the changes to the clue-control element specifically. Then I'd assume it would override any changes the CMS made and any rules within CLUE would override both of those because they'd be targeting elements inside of the clue-control.

Reply:
@scytacki Thanks. Yeah, it works when the rules are applied to the clue-control element specifically. The `.canvas .document-content` part wasn't needed.

I added a `clue-control` class name to the element to use as a selector. It didn't have to be an ID and since `EditableDocumentContent` currently only supports setting a class name, I just stuck with that.

I also shifted the `loading-box` CSS rules to custom-widget.scss. That seemed to make more sense after adding `clue-control` CSS rules to clue-control.scss. Looks like the loading box is a custom widget but not the clue control.